.gitignore

```diff
@@ -1,3 +1,4 @@
 *.swp
 schema/validate
 /bin
+vendor/
```

Comment:
nitpick: not a blocker necessarily, but this looks like an unrelated (and perhaps also unnecessary) change.

Reply:
Good point. @tariq1890 do you mind rolling back this change and the one in the sanity workflow?

Feel free to create a separate PR for these though.

Reply:
Sure I can remove this change, but I think we need the change in the sanity workflow to get the CI checks to pass